crates/aptos/src/move_tool/disassembler.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use move_command_line_common::files::{
+    MOVE_COMPILED_EXTENSION, MOVE_EXTENSION, SOURCE_MAP_EXTENSION,
+};
+use crate::common::types::{CliCommand, CliTypedResult};
+use std::{fs, path::Path};
+use move_binary_format::binary_views::BinaryIndexedView;
+use move_binary_format::CompiledModule;
+use move_binary_format::file_format::CompiledScript;
+use move_coverage::coverage_map::CoverageMap;
+use move_disassembler::disassembler::{Disassembler, DisassemblerOptions};
+use clap::Parser;
+use async_trait::async_trait;
+use move_bytecode_source_map::{mapping::SourceMapping, utils::source_map_from_file};
+use move_ir_types::location::Spanned;
+
+/// Disassemble the Move bytecode pointed to
+#[derive(Debug, Parser)]
+pub struct AptosDisassembler {
+    /// Skip printing of private functions.
+    #[clap(long = "skip-private")]
```

Comment:
I know this is just copy pasta, but it's a good chance for some clean up. With that in mind, let's just do this:
```suggestion
    #[clap(long)]
```
Same for all the other ones where the value for `long` just directly matches the arg name.